src/SprykerEco/Zed/AfterPay/Persistence/Propel/Schema/spy_after_pay.schema.xml

```diff
@@ -72,9 +72,9 @@
 
     <table name="spy_payment_after_pay_authorization">
         <column name="id_payment_after_pay_authorization" type="INTEGER" primaryKey="true" autoIncrement="true" />
-        <column name="order_reference" required="true" type="VARCHAR" />
-        <column name="id_checkout" required="false" type="VARCHAR"/>
-        <column name="id_reservation" required="false" type="VARCHAR"/>
+        <column name="order_reference" required="true" type="VARCHAR" size="45"/>
```

Comment:
I agree with https://github.com/spryker-eco/after-pay/pull/23/files#r707956031 comment, why it was closed @abitskil?

Reply:
Done similar to the https://github.com/spryker/spryker/pull/7477.